experiments/interpretation/dutchf3_patch/local/test.py

```diff
@@ -267,28 +267,42 @@ def _evaluate_split(
 
     n_classes = test_set.n_classes
 
+    if debug:
+        data_flow[split] = dict()
+        data_flow[split]['test_section_loader_length'] = len(test_set)
+        data_flow[split]['test_input_shape'] = test_set.seismic.shape
+        data_flow[split]['test_label_shape'] = test_set.labels.shape
+        data_flow[split]['n_classes'] = n_classes
+
+
     test_loader = data.DataLoader(test_set, batch_size=1, num_workers=config.WORKERS, shuffle=False)
 
     if debug:
+        data_flow[split]['test_loader_length'] = len(test_loader)
         logger.info("Running in Debug/Test mode")
-        test_loader = take(2, test_loader)
+        take_n = 2
+        test_loader = take(take_n, test_loader)
+        data_flow[split]['take_n_sections'] = take_n
+        pred_list, gt_list, img_list = [], [], []
+
 
     try:
         output_dir = generate_path(
-            f"debug/{config.OUTPUT_DIR}_test_{split}", git_branch(), git_hash(), config.MODEL.NAME, current_datetime(),
+            f"{config.OUTPUT_DIR}/test/{split}", git_branch(), git_hash(), config.MODEL.NAME, current_datetime(),
         )
     except:
-        output_dir = generate_path(f"debug/{config.OUTPUT_DIR}_test_{split}", config.MODEL.NAME, current_datetime(),)
+        output_dir = generate_path(f"{config.OUTPUT_DIR}/test/{split}", config.MODEL.NAME, current_datetime(),)
 
     running_metrics_split = runningScore(n_classes)
+   
 
     # evaluation mode:
     with torch.no_grad():  # operations inside don't track history
         model.eval()
-        total_iteration = 0
+        # total_iteration = 0
```

Comment:
I think the comment was that this wasn't needed - if it's not needed, should we just remove it?

Reply:
yes, I am pushing a commit that will remove this :)